eth/vm/forks/london/storage.py

```diff
@@ -0,0 +1,18 @@
+from eth_utils.toolz import partial
+
+from eth.vm.forks.berlin import constants as berlin_constants
+from eth.vm.forks.berlin.logic import (
+    GAS_SCHEDULE_EIP2929,
+    sstore_eip2929_generic,
+)
+
+
+SSTORE_CLEARS_SCHEDULE_EIP_3529 = GAS_SCHEDULE_EIP2929.sstore_reset_gas + \
+    berlin_constants.ACCESS_LIST_STORAGE_KEY_COST_EIP_2930
```

Comment:
nit: I'll do almost anything to avoid the `\` continuation. I prefer this style:
```suggestion
SSTORE_CLEARS_SCHEDULE_EIP_3529 = (
    GAS_SCHEDULE_EIP2929.sstore_reset_gas
    + berlin_constants.ACCESS_LIST_STORAGE_KEY_COST_EIP_2930
)
```